### PR TITLE
Reapply "add config for enabling tekton results dbssl"

### PR DESCRIPTION
**Pull request by enarha**
This reverts commit 91776680abc5935af19695150f709421429bda85 plus some tweaks to enable `verify-full` TLS verification.

### CODE REVIEW

**Comment by gabemontero** [comment]
@enarha let's create an infra-deployments test branch that points to your branch here

File: developer/openshift/apps/pipeline-service.sh [hunk 1]
```diff
@@ -56,6 +57,37 @@ EOF
   TEKTON_RESULTS_S3_PASSWORD="$(yq ".s3_password" "$tekton_results_credentials")"
 }
 
+generate_tekton_results_db_ssl_cert(){
```
[the remark this answers is not in the document]
Comment (enarha) [reply developer/openshift/apps/pipeline-service.sh:60]: We only need that locally as we use self signed certs. In stage/prod with AWS RDS we use the AWS Certs.

[the remark this answers is not in the document]
Comment (gabemontero) [reply developer/openshift/apps/pipeline-service.sh:60]: gotcha ... thanks for the clarification

**Comment by enarha** [comment]
> > @enarha let's create an infra-deployments test branch that points to your branch here
> 
> I'll do, even though I do not think it actually helps because it's still using the development stuff with the local Postgres.

> @enarha let's create an infra-deployments test branch that points to your branch here

I'll do, even though I do not think it actually helps because it's still using the development stuff with the local Postgres.

**Pull request merged**